src/test/java/io/gaia_app/encryption/NoOpEncryptionServiceIT.kt

```diff
@@ -0,0 +1,19 @@
+package io.gaia_app.encryption
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+//
+//@SpringBootTest(classes = [NoOpEncryptionService::class])
```

Comment:
Is the test class still useful ?

Reply:
No it is not.
And another PR will remove the NoOpEncryptionService, to better use inheritance.